lib/diagnostic.js

```diff
@@ -38,19 +47,34 @@ let owner = null;
 // Methods: vocalize
 
 // Replace null with your answer
-let pet = null;
+let pet = {
+  name: 'McFluff Boo',
+  species: 'monkey',
+  breed: 'capuchins',
+  vocalization: 'ow!',
+  vocalize: function () {
+    return this.vocalization;
+  },
+}
 
 // 4. Create a constructor function for pet objects, as described above. The
 // constructor function should be responsible for setting the values of `name`,
 // `species`, `breed`, and `vocalization`.
 
-const Pet = function() {
-  /* Your answer here */
+const Pet = function(name, species, breed, vocalization) {
+  this.name = 'McFluff Boo',
+  this.species = 'monkey',
+  this.breed = 'capuchins',
+  this.vocalization = 'ow!',
 };
 
 // 5. Attach a `vocalize` method to the Pet's prototype.
 
-/* your answer here */
+let bestFriend = new Pet ('McFluff Boo', 'monkey','capuchins','ow!')
+
+pet.prototype.vocalize = function () {
+  return this.vocalization;
+}
```

Comment:
`Pet.prototype` you wrote `pet` which will be undefined because the p is lowercase